matchzoo/tasks/ranking.py

```diff
@@ -1,5 +1,7 @@
 """Ranking task."""
 
+import keras
```

Comment:
do you need to import entire library?

Reply:
Yes in this case. Since ultimately we're going to have a  `layers` package in matchzoo, using `keras.layers` is better than using `layers` to avoid confusion.

Reply:
Then why not from keras import layers

Reply:
As I said, if you do `from keras import layers`, and use `layers` later, you can't tell if `layers` is come from keras or matchzoo. An alternative is `import keras.layers`, I can do that, but it's not that different.